packages/react-day-picker/src/components/Root/Root.test.tsx

```diff
@@ -121,3 +121,31 @@ describe('when "initialFocus" is set', () => {
     });
   });
 });
+
+describe('when using weekStartsOn prop', () => {
+  test('should start a week from Monday', () => {
+    const props: DayPickerProps = { weekStartsOn: 1 };
+
+    setup(props);
+
+    const firstDayOfTheWeek = screen
+      .getByRole('grid')
+      .getElementsByTagName('span')[1].textContent;
```

Comment:
not sure, if this is a good way to test this functionality...
Maybe you have some ideas, how can i rewrite the test?